rct229/ruleset_functions/get_baseline_system_types.py

```diff
@@ -64,14 +103,87 @@ def get_baseline_system_types(rmi_b):
             len(dict_with_terminal_units_and_zones[terminal_id]) == 1
             for terminal_id in terminal_unit_id_list
         ):
-            hvac_sys = is_baseline_system_7(
```

Comment:
Just a thought:

We could remove some of the repetitiveness here with something like:
```
# Put this up high in the file
baseline_system_type_checks = [is_baseline_system_1, is_baseline_system_2, ...]
```
Then here, we could use
```
sys_found = False
for sys_check in baseline_system_type_checks:
    hvac_sys = sys_check(rmi_b, hvac_b_id, terminal_unit_id_list, zone_id_list)
    if hvac_sys != HVAC_SYS.UNMATCHED:
        baseline_hvac_system_dict[hvac_sys].append(hvac_b_id)
        sys_found = True
        break
if not sys_found:
        # The error handling code goes here
```

Reply:
This is brilliant Charlie. I appreciate for letting me know about this.

This comment was addressed.